### ISSUE TITLE
Prep print ready files for Fundraising Team's envelopes

### ISSUE THREAD

**Issue by natalieworth** [MozillaFoundation/Design#645]
### Intro

Prep print-ready file templates for the envelope designs (Large envelop, no. 9 and no. 10 designs).

### Links
Redpen mockups are in here and have been reviewed/ approved by the fundraising team (Jesse, Stephanie, Hilary)
[Working files are in here](https://drive.google.com/open?id=1bmg_IZd3LWhTXoobxyPP83SqE4b6NQS2&authuser=natalie%40mozillafoundation.org&usp=drive_fs)

To do:
- [x] Print-ready files for No 9, 10, and large envelop
- [x]  Still waiting on feedback for the presentation folder. It's not needed yet, but if it's approved I can prep the file for when they are ready to print/ use it.

@kristinashu @phildexter Do you know if they'll need help coordinating with the printers?

**Comment by natalieworth** [comment]
[Print ready files are in here](https://drive.google.com/drive/folders/19TS2OlGDc3hV50FO-JKYXisLXsRssuDO?usp=sharing)

**Comment by natalieworth** [comment]
[Presentation print ready file is here](https://drive.google.com/drive/folders/15GtzAR63SlIaLtaap6L4OaEFZT3A1kE7?usp=sharing)

**Comment by natalieworth** [comment]
Sent the files to Jesse for printing.
